utils/docker-ce/files/dockerd.init

```diff
@@ -9,8 +9,65 @@ json_add_array_string() {
 	json_add_string "" "$1"
 }
 
+docker_bridge_add_config() {
+	local net="$1"
+
+	eval "$(ipcalc.sh "$net")"
+
+	# Add network interface
+	if ! uci -q get network.docker >/dev/null; then
+		uci -q batch >/dev/null <<EOF
+add network interface
```

Comment:
I'm not really a big fan of the "here document" redirection, it tends to make the code harder to read in my opinion.  
maybe we could just have a wrapper function that silences `uci` maybe
```sh
uci_quiet() {
    uci -q ${@} >/dev/null
}
uci_quiet add network interface
uci_quiet rename network.@interface[-1]=docker
...
```

Reply:
I think so too. I'll fix it.